### DISCUSSION TITLE
Updating email via JS Sdk doesn't work

### DISCUSSION THREAD

**Discussion by muhammed-kilinc** [pocketbase/pocketbase#2033] [Q&A] [answered]
Hello guys I can update the email of some user via admin ui but not with the sdk 
tested mails:
@gmail.com
@example.com 
@name.com

the code i use 

```
const res = await pb.collection("users").update(user.id, {
      ...user,
      ...newObject
    })
```

**Comment by ganigeorgiev** [comment, marked as answer]
Only admins and users with Manage API rule access can change users email directly.

Regular authenticated users need to call `pb.collection("users").requestEmailChange(newEmail)` (this will send an email with a confirmation link).
